pom.xml

```diff
@@ -84,6 +90,13 @@ Institute of Molecular Cell Biology and Genetics.</license.copyrightOwners>
 
 		<!-- NB: Deploy releases to the ImageJ Maven repository. -->
 		<releaseProfiles>deploy-to-imagej</releaseProfiles>
+		<jackrabbit-webdav.version>2.19.1</jackrabbit-webdav.version>
+		<apache-httpclient.version>4.5.6</apache-httpclient.version>
+		<apache-httpcore.version>4.4.11</apache-httpcore.version>
+
+		<!-- skip reproducible build enforcer during development -->
+		<imagej-updater.version>0.10.0-SNAPSHOT</imagej-updater.version>
+		<enforcer.skip>true</enforcer.skip>
```

Comment:
The commits built on this snapshot version will need to be rewritten before this PR can be merged.